package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "publicsuffixlist",
-  "version": "2.0.0",
-  "description": "Javascript utility to handle Public Suffix List from Mozilla Foundation",
+  "version": "3.0.0",
+  "description": "JavaScript utility to handle Public Suffix List from Mozilla Foundation",
   "main": "publicsuffixlist.js",
+  "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
```

Comment:
Can you add yourself as contributor in [`"contributors"` property](https://docs.npmjs.com/cli/v6/configuring-npm/package-json/#people-fields-author-contributors)?

Reply:
Done.